vault/router.go

```diff
@@ -236,8 +236,9 @@ func (r *Router) routeCommon(req *logical.Request, existenceCheck bool) (*logica
 	// Hash the request token unless this is the token backend
 	clientToken := req.ClientToken
 	switch {
-	case strings.HasPrefix(original, "auth/token/"):
-	case strings.HasPrefix(original, "cubbyhole/"):
+	case strings.HasPrefix(originalPath, "auth/token/"):
+	case strings.HasPrefix(originalPath, "sys/"):
```

Comment:
Generally speaking we do not want logical backends to have access to the client tokens, however, given that the logical system code lives in the `vault` package, where tokens are tossed around elsewhere (e.g. in the token store) it feels like it's an appropriate place for bare tokens to be available.

After all, many endpoints in the system backend take in bare token values as parameters.

I think when it was just the one endpoint it was like "maybe we shouldn't expose these for just the one endpoint" but now with multiple needs there's more of a use-case for doing so.